prog/main.go

```diff
@@ -29,11 +29,12 @@ var (
 	// set at build time
 	version = "dev"
 	// tokens to be elided when logging
-	serviceTokenFlag       = "service-token"
-	probeTokenFlag         = "probe.token"
-	kubernetesPasswordFlag = "probe.kubernetes.password"
-	kubernetesTokenFlag    = "probe.kubernetes.token"
-	sensitiveFlags         = []string{
+	serviceTokenFlag              = "service-token"
+	probeEcsClusterRegionFlag     = "probe.ecs.cluster.region"
```

Comment:
I wouldn't consider the ECS region sensitive enough to be hidden in logging output by default (for example, IPs aren't considered sensitive either). The hidden things are all passwords or authentication tokens.

Reply:
Done!